From 70679e219841733000bf5abfc2462872058a4042 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Pale=C4=8Dek?= <jpalecek@web.de>
Date: Thu, 20 Apr 2023 16:18:22 +0200
Subject: [PATCH 2/3] Remove dead code from FavoriteCollectionsModel

 This code has per search been always dead. On top of that, the for
 loop is faulty

(cherry picked from commit 56706add168cdc2a83eda49bfed7714a3ad01cc5)
---
 src/core/models/favoritecollectionsmodel.cpp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/core/models/favoritecollectionsmodel.cpp b/src/core/models/favoritecollectionsmodel.cpp
index 3ba7cec2c..835d8e277 100644
--- a/src/core/models/favoritecollectionsmodel.cpp
+++ b/src/core/models/favoritecollectionsmodel.cpp
@@ -152,13 +152,6 @@ public:
         }
     }
 
-    void clearReferences()
-    {
-        for (const Collection::Id &collectionId : std::as_const(referencedCollections)) {
-            dereference(collectionId);
-        }
-    }
-
     /**
      * Adds a collection to the favorite collections
      */
-- 
2.40.0

